### DISCUSSION TITLE
2006/10/03/%E5%9C%A8-labview-%E4%B8%AD%E5%AE%9E%E7%8E%B0-vi-%E7%9A%84%E9%80%92%E5%BD%92%E8%B0%83%E7%94%A8

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#119] [Announcements]
# 2006/10/03/%E5%9C%A8-labview-%E4%B8%AD%E5%AE%9E%E7%8E%B0-vi-%E7%9A%84%E9%80%92%E5%BD%92%E8%B0%83%E7%94%A8

LabVIEW 中使用递归调用不是很方便。不过递归并不是编程必须程序结构，任何需要使用递归调用的地方，都可以用循环结构来代替。但是在某些情况下，使用递归调用的确可以大大简化程序代码，对缩短编程时间、提高程序可读性都非常有帮助，所以学习一下递归的实现方法还是有好处的。

https://blog.qizhen.xyz/2006/10/03/%E5%9C%A8-labview-%E4%B8%AD%E5%AE%9E%E7%8E%B0-vi-%E7%9A%84%E9%80%92%E5%BD%92%E8%B0%83%E7%94%A8

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2009年08月12日 16:16 编辑
LV2009 可以直接递归，不在类中的VI也可以。

回复
l
2009年08月12日 15:50 编辑
现在LabVIEW2009终于可以递归了,不过是不是必须要使用类才可以呢?

回复
Qizhen
2008年05月23日 18:06 编辑
示例存放的位置已经被更新了

回复
huiyan
2007年09月4日 15:26 编辑
阮前辈,这个怎么下载不了啊  
http://ruanqizhen.googlepages.com/Factorial.vi ————递归

回复
斌
2007年02月8日 10:16 编辑
我以为LV不能递归的……，还专门用stack来解决hanoi问题…… 看了你的文章我要再重写一下程序，用两者比较看看！
谢谢！！
